pandas/tests/groupby/test_index_as_string.py

```diff
@@ -67,50 +66,3 @@ def test_grouper_index_level_as_string_series(series, levels):
     # Compute and check result
     result = series.groupby(levels).mean()
     assert_series_equal(result, expected)
-
-
-@pytest.mark.parametrize('key_strs,key_groupers,level_groupers', [
-    ('inner',  # Index name
-     pd.Grouper(key='inner'),
-     pd.Grouper(level='inner'),
-     ),
-    (['inner'],  # List of index name
-     [pd.Grouper(key='inner')],
-     [pd.Grouper(level='inner')]
-     ),
-    (['B', 'inner'],  # Column and index
```

Comment:
maybe just change to have these check for raising? or too duplicative?

Reply:
Too duplicative --> it raises.